tensorflow/python/ops/array_ops.py

```diff
@@ -490,23 +490,32 @@ def setdiff1d(x, y, index_dtype=dtypes.int32, name=None):
 def broadcast_dynamic_shape(shape_x, shape_y):
   """Computes the shape of a broadcast given symbolic shapes.
 
-  When shape_x and shape_y are Tensors representing shapes (i.e. the result of
-  calling tf.shape on another Tensor) this computes a Tensor which is the shape
-  of the result of a broadcasting op applied in tensors of shapes shape_x and
-  shape_y.
-
-  For example, if shape_x is [1, 2, 3] and shape_y is [5, 1, 3], the result is a
-  Tensor whose value is [5, 2, 3].
+  When `shape_x` and `shape_y` are Tensors representing shapes (i.e. the result
+  of calling tf.shape on another Tensor) this computes a Tensor which is the
+  shape of the result of a broadcasting op applied in tensors of shapes
+  `shape_x` and `shape_y`.
 
   This is useful when validating the result of a broadcasting operation when the
   tensors do not have statically known shapes.
 
+  Example:
+
+  >>> shape_x = (1, 2, 3)
+  >>> shape_y = (5, 1, 3)
+  >>> tf.broadcast_dynamic_shape(shape_x, shape_y)
+  <tf.Tensor: id=..., shape=(3,), dtype=int32, numpy=array([5, 2, 3], \
```

Comment:
We recently removed id=... from tensor printing

Reply:
Okay, added that change. :+1: